tasks/skylab/FastqProcessing.wdl

```diff
@@ -10,7 +10,8 @@ task FastqProcessing {
     String sample_id
 
     # runtime values
-    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v0.3.11"
+    String docker = "quay.io/humancellatlas/fastq-process:latest"
```

Comment:
Why did we switch from the sctools docker? Is the dockerfile to build this image checked in somewhere?

Reply:
We need to have the review done before we add the new docker. I invited you as a reviewer for that PR. All changes are cosmetic (more information is printed about the inputs)